NUService.js

```diff
@@ -40,14 +48,15 @@ export default class NUService extends NUObject {
                 headerPage: headers.headerPage,
                 headerPageSize: headers.headerPageSize,
                 password: null,
-                rootURL,
+                rootURL: url,
```

Comment:
```
rootURL: rootURL ? rootURL : `${protocol}://${hostname}${port ? ":" + port}/${RESTRoot}/${RESTResource}${APIVersion}`
```

Reply:
in our previous discussion, we agreed that if rootURL is passed same will be used, else rootURL will be constructed using the object passed in.
as per this, if object is passed, `url` is already constructed using the passed object. why again do this?
only change I see is port being optional, line 38 can be updated to take care of this.

Reply:
done, url is initialized according to out latest discussion.